### PR TITLE
Aliases for pickpocket npcs

### PR DESCRIPTION
**Pull request by TastyPumPum**
Added aliases for various pickpockets, unable to add any to Pollnivnian Bandit until the oldschooljs PR is merged.  As it is using the wrong ID currently. 

https://github.com/oldschoolgg/oldschooljs/pull/275

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/commands/Minion/pickpocket.ts [hunk 1]
```diff
@@ -65,7 +65,10 @@ export default class extends BotCommand {
 			quantity = null;
 		}
 
-		const pickpocketable = Pickpocketables.find(npc => stringMatches(npc.name, name));
+		await msg.author.settings.sync(true);
```
Comment (gc) [inline src/commands/Minion/pickpocket.ts:68]: why did you add this?

**Review by gc — CHANGES_REQUESTED** [review]

File: src/commands/Minion/pickpocket.ts (hunk 1 above)
Comment (TastyPumPum) [reply src/commands/Minion/pickpocket.ts]: Apologies, copied it over accidentally and everything worked fine in testing so didn't pick up on it
